lib/core/unittest/CConcurrentWrapperTest.cc

```diff
@@ -9,13 +9,14 @@
 #include <core/CConcurrentWrapper.h>
 #include <core/CLogger.h>
 #include <core/CMemoryUsage.h>
+#include <core/CStaticThreadPool.h>
 
 #include <boost/bind.hpp>
 #include <boost/ref.hpp>
-#include <boost/threadpool.hpp>
 
 #include <algorithm>
 #include <chrono>
+#include <iomanip>
```

Comment:
Is this needed?

Reply:
It's needed for e.g. [this](https://github.com/elastic/ml-cpp/blob/master/lib/core/unittest/CConcurrentWrapperTest.cc#L58). I think it must have been coming in via the old thread pool header in the past.